sklearn/utils/optimize.py

```diff
@@ -39,8 +40,37 @@ def _line_search_wolfe12(f, fprime, xk, pk, gfk, old_fval, old_old_fval, **kwarg
     """
     ret = line_search_wolfe1(f, fprime, xk, pk, gfk, old_fval, old_old_fval, **kwargs)
 
+    if ret[0] is None:
```

Comment:
I think that this change should be reflected in an entry in the changelog as well.

Reply:
You been more detail than just
> ...have much better convergence for solvers `"lbfgs"` and `"newton-cg"`

I can add a changelog later today.

Reply:
I meant to have an entry in the `sklearn.linear_model` section specifically tagged as an enhancement.

Reply:
Do you mean a 2nd entry or add that detail to the existing entry (of this PR)? The latter?